webpack.config.js

```diff
@@ -35,7 +35,7 @@ module.exports = (shell = {}) => {
     mode: IS_PROD ? 'production' : 'development',
     devtool: IS_PROD ? 'source-map' : 'cheap-eval-source-map',
     resolve: {
-      extensions: ['.js', '.jsx'], // resolves `import '../Foo'` to `../Foo/index.jsx`
+      extensions: ['.js', '.jsx', '.json'], // resolves `import '../Foo'` to `../Foo/index.jsx`
```

Comment:
I dont think this is needed, and risks bundling json files in the application

Reply:
Talked IRL, reverted.